subsys/shell/Kconfig

```diff
@@ -251,6 +251,13 @@ config SHELL_AUTOSTART
 	help
 	  If enabled, shell will be automatically started.
 
+config SHELL_RETURN_VALUE
+	bool "Command to retrieve return value"
```

Comment:
it should depend on `SHELL_CMDS`. Could also be renamed to `SHELL_CMDS_RETURN_VALUE` as there are already `SHELL_CMDS_RESIZE` or `SHELL_CMDS_SELECT`.